phpstan.neon

```diff
@@ -1,3 +1,5 @@
+includes:
+    - vendor/phpstan/phpstan/conf/config.level5.neon
```

Comment:
Why is this better than just passing the level from CLI parameters?

Reply:
it might've changed recently, but there's no cli parameter for the max level, which currently aliases to `-l7`, but may change in future.

there's also the `phpstan/phpstan-strict-rules` ruleset, as well as any custom configurations that may be required in future.

Reply:
Yes, it's possible to [use max in CLI](https://github.com/phpstan/phpstan#rule-levels).
> You can also use `--level max` as an alias for the highest level.

Reply:
well colour me corrected :P

pretty sure that importing rules still stands?

Reply:
Yep, other rules/extensions should be imported in the config.